variables.tf

```
@@ -426,6 +426,12 @@ variable "kafka_topic_name" {
   default     = ""
 }
 
+variable "s3_bucket_expiry" {
```

```suggestion
variable "s3_bucket_expiry_days" {
```
Is it possible to rename this so it's more descriptive?

---

I would suggest `s3_object_expiration_days`

---

That's even better, yes.